docs/messaging-implementation.md

```diff
@@ -22,6 +22,7 @@ point uportal-app-framework to your desired feed.
             "expireDate": "2017-08-02",
             "featureImageUrl": null,
             "priority": "high",
+            "recurrance": true,
```

Comment:
Spelling mismatch between here, the property in `services.js`, and the documentation below.